cmd/tendermint/commands/inspect.go

```diff
@@ -0,0 +1,68 @@
+package commands
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/spf13/cobra"
+
+	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tendermint/inspect"
+	"github.com/tendermint/tendermint/state"
+	"github.com/tendermint/tendermint/state/indexer/sink"
+	"github.com/tendermint/tendermint/store"
+	"github.com/tendermint/tendermint/types"
+)
+
+// InspectCmd is the command for starting an inspect server.
+var InspectCmd = &cobra.Command{
+	Use:   "inspect",
+	Short: "Run an inspect server",
+
```

Comment:
can we have a long description as well.

Reply:
Mind giving me a bit more info on when this mode will be useful? I see in the github issue that tendermint will panic with a 'divergent hash'. Which hash is this exactly and are there maybe other times this may be helpful so that I can include them in the `Long` description?

`.$HOME/.tendermint/data` is the default path do the data dir.

Reply:
Added a first pass at the Long description. Let me know what may need to change.